server/protocols/webadmin-integration-test/distributed-webadmin-integration-test/src/test/java/org/apache/james/webadmin/integration/rabbitmq/ConsistencyTasksIntegrationTest.java

```diff
@@ -418,8 +483,8 @@ void recomputeQuotasShouldSolveNotingWhenNoInconsistencies(GuiceJamesServer serv
                     .whenQueryStartsWith(updatedQuotaQueryString));
 
         probe.appendMessage(BOB.asString(), inbox,
-            new ByteArrayInputStream("Subject: test\r\n\r\ntestmail".getBytes(StandardCharsets.UTF_8)), new Date(),
-            !IS_RECENT, new Flags(Flags.Flag.SEEN));
+            new ByteArrayInputStream(MESSAGE_CONTENT_AS_BYTES), DATE,
+            !IS_RECENT, FLAGS);
```

Comment:
Can we use `public ComposedMessageId appendMessage(String username, MailboxPath mailboxPath, MessageManager.AppendCommand appendCommand)` instead?